cxx-qt-gen/src/parser/parameter.rs

```diff
@@ -0,0 +1,14 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use syn::{Ident, Type};
+
+/// Describes a Parameter for a QObject
+pub struct ParserParameter {
+    /// The [syn::Ident] of the parameter
```

Comment:
this level of documentation is probably a bit overkill.
No need to state the obvious :)

Reply:
Left for now, this might become more complicated later :-)